### DISCUSSION TITLE
✅ ♿ option to disable anchor-link scroll JavaScript

### DISCUSSION THREAD

**Discussion by m1ga** [elementor/elementor#25795] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

All anchor links in a navigation have a JavaScript function that smoothly scrolls to the target position. This prevents setting a custom scroll-padding with CSS. 

There is an old ticket from 2019 https://github.com/elementor/elementor/issues/8808 with some more infos. As you want to use discussions so that people can upvote those requests I'm putting it in here.

### Describe the solution you'd like.

Add a setting so we can disable the JS anchor link scroll in https://github.com/elementor/elementor/blob/4f2417f6b6ee086d404df8a30edf51c3ea4bedd0/assets/dev/js/frontend/utils/anchors.js#L28C2-L28C19

or use
```css
html {
  scroll-behavior: smooth;
}
```
to handle it with CSS instead. For backwards compatibility the option would be better

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by littlefoxsox** [comment]
+1
I would also very much appreciate the ability to disable smooth-scroll. Currently, this "feature" is getting in the way of desired functionality (I want users to be able to snap to anchor links without a smooth-scroll effect every time). I've tried various custom codes and java script tutorials but they don't seem to work on the current version of Elementor (or something else is interfering) .

**Reply by nicholaszein** [reply]
Hi @littlefoxsox!

This is already the native behavior of Elementor since version 3.25.

Thanks for commenting though, you made me find this missing request. 🙂

**Comment by nicholaszein** [comment]
Solved in version 3.25. 🙂
